test/integration/scheduler/scheduler_test.go

```diff
@@ -749,3 +749,96 @@ func TestPDBCache(t *testing.T) {
 		t.Errorf("No PDB was deleted from the cache: %v", err)
 	}
 }
+
+// TestSchedulerEvents tests that scheduler receives events and updates its cache when
+// pods are scheduled by other schedulers.
+func TestSchedulerEvents(t *testing.T) {
```

Comment:
Please don't call this "events", you nearly gave me a heart-attack here.  

No controller/scheduler in the system can watch events, it would implode the system 

/cc @kubernetes/sig-scalability-misc

Reply:
Haha. These are informer events, not "those" events! :smile: